lib/tags/profile-settings.tag

```diff
@@ -374,13 +380,15 @@
                             self.profile.about = bio;
                             self.refs.bio.classList.remove('graphjs-error');
                             self.refs.bio.classList.add('graphjs-success');
+                            self.refs.bio.setAttribute('data-savesuccesstext', i18n.saveSuccessText);
```

Comment:
Please remove this change. self.refs.bio refers to an input element and input elements do not accept ::before or ::after pseudo-elements.

Reply:
Same as above